### PR TITLE
Configuring Heroku to use Nginx in front of gunicorn

### PR DESCRIPTION
**Pull request by dryan**

### CODE REVIEW

**Comment by dryan** [comment]
I've got this working on a preview app. Currently I've set the rate to 5 requests per minute plus a burst of an additional 20. Totally open to discussing that limit, it was just easy to test.

**Comment by iandees** [comment]
What's left on this, @dryan?

File: config/nginx.conf.erb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	accept_mutex on;
+	worker_connections 1024;
+}
+
+http {
+  gzip on;
+  gzip_comp_level 2;
+  gzip_min_length 512;
+
+	server_tokens off;
```
Comment (iandees) [inline config/nginx.conf.erb]: Mixed spaces/tabs in here if we care.

**Comment by dryan** [comment]
Would like another set of eyes testing it. And for us to figure out what the actual limits should be.

File: config/nginx.conf.erb (hunk 1 above)
Comment (dryan) [reply config/nginx.conf.erb]: I'll fix them

**Pull request merged**